File: 9. Palindrome Number/solution.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if (x < 0):
            return False

        result = x
        temp = 0

        while (result != 0):
            temp = temp * 10 + result % 10
            result = result // 10

        if (temp == x):
            return True

        return False

File: 9. Palindrome Number/solution.php
<?php
class Solution {

    /**
     * @param Integer $x
     * @return Boolean
     */
    function isPalindrome($x) {
        if ($x < 0)
            return false;

        $sum = 0;
        $a = $x;
        while(0 != $a) {
            $sum = $sum * 10 + ($a % 10);
            $a = intdiv($a, 10);
        };

        return ($x == $sum);
    }
}